src/Math/Angle.cs

```diff
@@ -44,7 +41,7 @@ public static Angle Between(Vector2 from, Vector2 to)
         {
             float perpDot = from.Y * to.X - from.X * to.Y;
 
-            return Angle.FromRadians(Mathf.Atan2(perpDot, Vector2.Dot(from, to)));
+            return FromRadians(Mathf.Atan2(perpDot, Vector2.Dot(from, to)));
```

Comment:
I think it makes sense to re-add the class names here. If we even want to refactor this, we should do it project wide in a separate pull request.